velox/vector/fuzzer/tests/VectorFuzzerTest.cpp

```diff
@@ -688,6 +688,24 @@ TEST_F(VectorFuzzerTest, fuzzRowChildrenToLazy) {
   ASSERT_TRUE(wrappedRow->childAt(1)->as<LazyVector>()->isLoaded());
 }
 
+TEST_F(VectorFuzzerTest, constantOpt) {
```

Comment:
naming: do not abbreviate: constantRatio

However, this test tests both constant and dictionary ratio, hence the name doesn't reflect the test logic. It might be encodingRatios, but then the test only tests zero ratios. You may need to extend the test to verify multiple combinations: both ratios zero, only one ratio is zero but < 1, both non-zero but < 1, both are 1, one is 1, etc.

Reply:
I've updated the PR, added a fuzzInputFlatRow API, and removed the new opt and this ut.